tsfile/src/main/java/org/apache/iotdb/tsfile/read/reader/page/PageReader.java

```diff
@@ -109,25 +111,48 @@ private BatchData getAllPageData() throws IOException {
     while (timeDecoder.hasNext(timeBuffer)) {
       long timestamp = timeDecoder.readLong(timeBuffer);
 
-      pageData.putTime(timestamp);
       switch (dataType) {
         case BOOLEAN:
-          pageData.putBoolean(valueDecoder.readBoolean(valueBuffer));
+          boolean aBoolean = valueDecoder.readBoolean(valueBuffer);
```

Comment:
I think we should take these examination type code (boolean, int32, int64, float, double, etc) into single methods.
Maybe we can do it after the `fix_sonar` pr.

Reply:
Yeah，I've seen that PR and I will leave this to later merge.